packages/progress-bar/src/ProgressBar.ts

```diff
@@ -121,13 +143,12 @@ export class ProgressBar extends SizedMixin(SpectrumElement) {
             ) {
```

Comment:
The actual test here likely needs to change in order to support the new path to labeling this element with the `textContent`. Tests for the various contexts that we'd expect this to warn user with should look like https://github.com/adobe/spectrum-web-components/blob/main/packages/progress-circle/test/progress-circle.test.ts#L81-L103 or so.